src/auth/channel.ts

```diff
@@ -42,6 +42,25 @@ export const createWssChannel = async (options: ChannelOptions): Promise<Channel
   }
 }
 
+const waitForRootDid = async (username: string): Promise<string | null> => {
+  return new Promise((resolve) => {
+    const maxTries = 3
+    let tries = 0
+    let rootDid = null
+
+    const rootDidInterval = setInterval(async () => {
+      rootDid = await await did.root(username).catch(() => null)
```

Comment:
Can we remove one of the `await`s here? (This was there before your PR. Don't think we need it.)